runtime/src/bank.rs

```diff
@@ -1436,10 +1436,22 @@ impl Bank {
     /// calculation and could shield other real accounts.
     pub fn verify_snapshot_bank(&self) -> bool {
         self.purge_zero_lamport_accounts();
-        self.rc
+        if !self
+            .rc
             .accounts
             .verify_hash_internal_state(self.slot(), &self.ancestors)
-            && !self.has_accounts_with_zero_lamports()
+        {
+            info!(
```

Comment:
+1 for more detailed message!

Eventually `verify_snapshot_bank` causes termination, so can `warn!` be more appropriate?

Reply:
Well, I'm thinking if you are running a test which is designed to hit this, then it is not a problem and in the case that we panic, the error is already obvious from that, this just gives some extra detail.

Reply:
I see!